examples/expect-extend/toBeWithinRange.ts

```diff
@@ -8,8 +8,8 @@
 import {expect} from '@jest/globals';
 import type {MatcherFunction} from 'expect';
 
-const toBeWithinRange: MatcherFunction<[floor: number, ceiling: number]> =
-  function (actual: unknown, floor: unknown, ceiling: unknown) {
+const toBeWithinRange: MatcherFunction<[floor: unknown, ceiling: unknown]> =
+  function (actual, floor, ceiling) {
```

Comment:
Cleaning up. I remember my thinking that `expect.extend` could grab types from `MatcherFunction` generics. Not sure if that was good idea. It involves lots of complexity (also in documentation), but has rather limited usage. For instance, does not work if `expect` is not imported after calling `expect.extend`. Fine for something small, but does not work with `setupFilesAfterEnv`. Better to stick with the declaration – single solution which works in many situations.